assembly.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, n_start, n_trimmed, n_rmdup, n_output, n_subsamp, n_unpaired_
 
 def trim_rmdup_subsamp_reads(inBam, clipDb, outBam, n_reads=100000):
```

Comment:
Once you fix trimmomatic, apparently not! It's kind of beautiful, especially watching the little report at the end with all the counts.